### PR TITLE
Fix jump aye

### PR DESCRIPTION
**Pull request by mattiejas**

### CODE REVIEW

File: engine/physics/World.cpp [hunk 1]
```diff
@@ -3,6 +3,7 @@
 #include <Box2D/Dynamics/b2World.h>
 
 #include <chrono>
+#include <iostream>
```
Comment (FabioGNR) [inline engine/physics/World.cpp]: iostream is not used?

File: game/CMakeLists.txt [hunk 2]
```diff
@@ -12,10 +12,11 @@ set(GAME_SRC
         states/BaseMenuState.h
         exceptions/NoPlayersFoundException.h
         exceptions/MissingCharacterSpawnException.h
-        )
+        listeners/JumpContactListener.cpp listeners/JumpContactListener.h)
```
Comment (FabioGNR) [inline game/CMakeLists.txt]: Newlines

File: game/CMakeLists.txt [hunk 3]
```diff
@@ -56,7 +57,9 @@ set(SYSTEM_SRC
         systems/items/ReverseGravitySystem.cpp
         systems/items/ReverseGravitySystem.h
         systems/LifeSystem.cpp
-        systems/LifeSystem.h)
+        systems/LifeSystem.h
+        systems/JumpSystem.cpp 
+        systems/JumpSystem.h)
```
Comment (FabioGNR) [inline game/CMakeLists.txt]: ) on newline?

File: game/systems/MovementSystem.h [hunk 4]
```diff
@@ -2,6 +2,7 @@
 
 #include <engine/ecs/System.h>
 #include <engine/ecs/World.h>
+#include <engine/physics/ContactListener.h>
```
Comment (FabioGNR) [inline game/systems/MovementSystem.h]: unused include?

File: engine/physics/World.cpp [hunk 5]
```diff
@@ -156,5 +168,57 @@ namespace physics {
...
+        }
+
+        std::for_each(physicsWorld->m_contactListeners.begin(),
+            physicsWorld->m_contactListeners.end(),
+            [&body1, &body2](auto& cl) { cl->endContact(Contact{ body1, body2 }); });
+    }
+
+    void World::WorldImpl::PreSolve(b2Contact* contact, const b2Manifold* oldManifold)
+    {
+        b2ContactListener::PreSolve(contact, oldManifold);
```
Comment (FabioGNR) [inline engine/physics/World.cpp]: Is this intended? Seems like generated code left behind?

File: .clang-format [hunk 6]
```diff
@@ -1 +1,2 @@
-BasedOnStyle: WebKit
\ No newline at end of file
+BasedOnStyle: WebKit
+SpaceBeforeCpp11BracedList: false
```
Comment (notzain) [inline .clang-format]: gitignore this file if you make changes

**Comment by FabioGNR** [comment]
All requests fixed, if the builds succeed I'll approve. Not sure if the deletion of clang-format is the right move, but Travis will let us know.

**Pull request merged**